org-formation/organization.yaml

```diff
@@ -356,6 +365,8 @@ Organization:
         <<: !Include ./_default_org_tags.yaml
         Department: CompOnc
         Project: iAtlas
+        AccountOwner: undecided@sagebase.org
```

Comment:
I know its documented that Andrew Lamb is the System owner, but I'm unsure how much involvement he actually has.  Would need more discussion internally first, but Brian is out.